### DISCUSSION TITLE
My smartphone doesn't take photos smaller than 2 MB. How can I send the documents?

### DISCUSSION THREAD

**Discussion by ricarditt0** [community/community#141734] [GitHub Education] [answered]
### Hi everyone,

i can't take photos smaller than 2 MB and only frontal camera is avalible to photograph the documents on the page so .How i can send my documents?

**Comment by Iced-Coded** [comment, marked as answer]
You can use FFMPEG to compress your images, or... One of the "compressor" websites, though, not recommended due to privacy reasons.
